app/search_builders/hyrax/collection_member_search_builder_decorator.rb

```diff
@@ -0,0 +1,39 @@
+# OVERRIDE: Hyrax 3.4.1 adds filesets to search to allow full text search results
+
+# frozen_string_literal: true
+
+module Hyrax
+  module CollectionMemberSearchBuilderDecorator
+    Hyrax::CollectionMemberSearchBuilder.default_processor_chain += [:show_works_or_works_that_contain_files]
+
+    # These methods include the filesets in the search results
+    def show_works_or_works_that_contain_files(solr_parameters)
```

Comment:
@labradford can you explain the name of this method?

Reply:
That's what it was already named in Hyrax here: https://github.com/samvera/hyrax/blob/fd2f525007bedb11a067cec33250a826556f883f/app/search_builders/hyrax/catalog_search_builder.rb#L39